tests/e2e/backup_restore_suite_test.go

```diff
@@ -186,7 +261,7 @@ var _ = Describe("AWS backup restore tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			// wait for backup to not be running
-			Eventually(IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*12, time.Second*10).Should(BeTrue())
+			Eventually(IsBackupDone(dpaCR.Client, namespace, backupName), timeoutMultiplier*time.Minute*20, time.Second*10).Should(BeTrue())
```

Comment:
Since this is data mover test, would it be possible to also check if VSB is completed here?

Reply:
Velero backup status should be reliable. VSB status should really only rarely be checked when things go wrong.

Reply:
Let's leave it out. Could add on failure. Follow up todo?

Reply:
Actually VSB/VSR is already checked on failure

Reply:
*logged on failure.

Reply:
right here https://github.com/openshift/oadp-operator/pull/863/files#diff-a9f567f9b6083a8f8e52d9318806042225c729fa99bea2354e148eead33bdf9fR119-R124